User Guides/18-Deleting a deployment.md

```diff
@@ -0,0 +1,21 @@
+# Deleting the Deployment
+
+Since the deployment provisions resources in your subscription, you will be billed for the usage. If you want to discard use of the deployment, we recommend deleting the resources that were provisioned so that you are not billed for the usage.
+Since deleting the deployment will also remove the data in your physical data warehouses, please consider backing up your data before deleting the deployment.
+
+In order to delete the deployment, please do the following:
+
+## Delete the resource group
+1. On the [Azure Portal](https://portal.azure.com), search for your resource group. The resource group is the name of the deployment that you provided when setting up the deployment.
+2. Then click on 'Delete resource group' and follow the prompts to delete the resource group.
+3. If you have locks that are preventing deletion of the resource groups, you can find the locks in the Locks pane for your resource group and delete them first prior to deleting the resource group.
```

Comment:
Run on sentence.

Reply:
Run on sentence? What does that mean? Please elaborate. I don't get it.

Reply:
It's just a long overly complex sentence. Try to cleanup.